packer/scripts/extras/redis.sh

```diff
@@ -0,0 +1,228 @@
+# COPY healthcheck.sh - Create all necessary files
```

Comment:
To też wygląda jak kod dla kontenera nie VM

Reply:
skrypty kontenerowe zostały usunięte